docs/user/02-operation-guides/operations/02-10-check-if-sidecar-injection-is-enabled.md

```diff
@@ -0,0 +1,82 @@
+# Check if you have Istio sidecar proxy injection enabled
+
+## Check if sidecar injection is enabled in the Pod's Namespace
+
+<!-- tabs:start -->
+
+#### **kubectl**
+Run the following command to check if the Pod's Namespace is labeled with `istio-injection=enabled`:
+
+    ```bash
+    kubectl get namespaces {NAMESPACE} -o jsonpath='{ .metadata.labels.istio-injection }'
+    ```
+If the command does not return `enabled`, the sidecar injection is disabled in this Namespace.
+
+#### **Kyma Dashboard**
+
+1. Go to the Pod's Namespace.
+2. Verify if the `Labels` section contains `istio-injection=enabled`. If the section doesn't contain the label, the sidecar injection is disabled in this Namespace.
+   Here's an example of a Namespace where the Istio sidecar proxy injection is enabled:
+   ![Namespace with enabled istio sidecar injection](../../../assets/namespace-with-enabled-istio-sidecar.svg)
+
+<!-- tabs:end -->
+
+## Check if sidecar injection is enabled for the Pod's Deployment
+
+<!-- tabs:start -->
+
+#### **kubectl**
+
+Run the following command:
+
+    ```bash
+    kubectl get deployments {DEPLOYMENT_NAME} -n {NAMESPACE} -o jsonpath='{ .spec.template.metadata.labels }'
+    ```
+Sidecar injection is disabled if the output does not contain the `sidecar.istio.io/inject:true` line.
+
+#### **Kyma Dashboard**
+
+1. Select the Namespace of the Pod's Deployment.
+2. On the left-hand side sidebar, find the **Workloads** section and select **Deployments**.
```

Comment:
I'd avoid describing the UI in detail; because it might change and then you'll have to adjust the instructions

```suggestion
2. In the **Workloads** section, select **Deployments**.
```